dl4j-distributed-training-examples/scripts/patentExamplePreproc.sh

```diff
@@ -31,12 +31,12 @@ MINIBATCH=32                                    #Minibatch size for preprocessed
 
 #Other variables. Don't modify these
 SCRIPTDIR=$(dirname "$0")
-JARFILE=${SCRIPTDIR}/../target/dl4j-spark-1.0.0-beta7-bin.jar
+JARFILE=${SCRIPTDIR}/../target/dl4j-spark-1.0.0-beta6-bin.jar
```

Comment:
Please also check all script files for beta6 <-> beta7 differences.

Reply:
Done